src/components/Wrapper/index.js

```diff
@@ -31,6 +31,7 @@ const Wrapper = styled.div`
       display: flex;
       justify-content: center;
     `};
+  ${({ stretch }) => stretch && `min-height: 100vh;`};
```

Comment:
Since there’s no content apart from the loader, we need to stretch the wrapper so that the loader is rendered correctly.

Reply:
👍